client/data/domains/nameservers/use-domain-nameservers-query.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+
+const useDomainNameserversQuery = ( domainName, queryOptions = {} ) =>
```

Comment:
Nit: What's the protocol for query options when we don't need them? I'm personally a fan of the YAGNI principle, which suggests not adding anything that we have no guarantee we're going to use. Meaning, if we don't need this at this time, we can just remove it.

Reply:
I think it boils down to whether we want to generally make the `useQuery` API available to consumers of our custom query hooks. In this particular case it's unlikely that we need it because `useDomainNameserversQuery` is only used at one place. I agree with the your sentiment and am fine with removing it.

Reply:
fixed with https://github.com/Automattic/wp-calypso/pull/53657/commits/a373cf1cb96bda8117ca9b5838ffdbe4b0182152